js_modules/dagit/packages/core/src/ui/TimeFromNow.tsx

```diff
@@ -1,18 +1,20 @@
 import {Tooltip} from '@dagster-io/ui';
-import moment from 'moment-timezone';
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime';
 import * as React from 'react';
 
 import {Timestamp} from '../app/time/Timestamp';
 
+dayjs.extend(relativeTime);
 const TIME_FORMAT = {showSeconds: true, showTimezone: true};
 
-export const TimeFromNow: React.FC<{unixTimestamp: number}> = ({unixTimestamp}) => {
+export const TimeFromNow: React.FC<{unixTimestamp: number}> = React.memo(({unixTimestamp}) => {
   return (
     <Tooltip
       placement="top"
       content={<Timestamp timestamp={{unix: unixTimestamp}} timeFormat={TIME_FORMAT} />}
     >
-      {moment.unix(unixTimestamp).fromNow()}
+      {dayjs(unixTimestamp * 1000).fromNow()}
```

Comment:
Only use case of dayjs so far.

Reply:
Nice! Glad we did a good job of getting this relative time + tooltip everywhere.